testrail/testrail.rb

```diff
@@ -0,0 +1,106 @@
+#
+# TestRail API binding for Ruby (API v2, available since TestRail 3.0)
+#
+# Learn more:
+#
+# http://docs.gurock.com/testrail-api2/start
+# http://docs.gurock.com/testrail-api2/accessing
+#
+# Copyright Gurock Software GmbH. See license.md for details.
+#
+
+require 'net/http'
+require 'net/https'
+require 'uri'
+require 'json'
+
+module TestRail
+	class APIClient
+		@url = ''
+		@user = ''
+		@password = ''
+
+		attr_accessor :user
+		attr_accessor :password
+
+		def initialize(base_url)
+			if !base_url.match(/\/$/)
```

Comment:
'unless' is the same as 'if !'
when you don't have else block it's better to use unless